views/register.mustache

```diff
@@ -1,26 +1,56 @@
-<html>
+<!DOCTYPE html>
+<html lang = en>
     <head>
-        <title>Register</title>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>LoginPage</title>
```

Comment:
Shouldn't this be "Register" or something like that?

Reply:
Yep, I forgot to change the titles for the other pages.